caas/kubernetes/provider/utils/labels.go

```diff
@@ -38,38 +38,40 @@ func HasLabels(src, has labels.Set) bool {
 	return true
 }
 
-// IsLegacyModelLabels checks to see if the provided model is running on an older
+const ErrBadModelLabels errors.ConstError = "bad model labels"
```

Comment:
comment

Reply:
Could this be "unexpected model labels" ?
They are not bad as such, just not what Juju expects.